Lib/tarfile.py

```diff
@@ -2339,10 +2325,8 @@ def _load(self):
         """Read through the entire archive file and look for readable
            members.
         """
-        while True:
-            tarinfo = self.next()
-            if tarinfo is None:
-                break
+        while (tarinfo := self.next()) is not None:
+            pass
```

Comment:
This is another controversial change that I like.  The first one looks like it is doing some sort of work to select or filter; the 2nd makes it more clear that it is just skipping through (but touching and therefore loading?) all the self.next() chunks.  

That said, the tarfile temp name is not used again.  It might be even more clear as

while self.next() is not None: pass